packages/interpreter/src/interpreters/bear/plugins/backlinksPlugin.test.ts

```diff
@@ -0,0 +1,102 @@
+import mdast from "mdast";
```

Comment:
all files in the `packages/interpreter/src/interpreters/bear/plugins` directory have been directly ported from the vendored `incremental-thinking` package. I thought moving these over might more sense then trying to vendor the package here again.

Reply:
Yes, I think that's the right call. I don't think that other package is going anywhere.